src/adapter/activateDaffodilDebug.ts

```diff
@@ -272,6 +346,64 @@ export function activateDaffodilDebug(
     )
   )
 
+  context.subscriptions.push(
```

Comment:
Concern: I don't know why these functions need to be registered as user-available commands. Can't they just be functions used by the extension itself?